### ISSUE TITLE
Configure to use only front camera

### ISSUE THREAD

**Issue by shobhik** [Drjacky/ImagePicker#35]
**Is your feature request related to a problem? Please describe.**
When starting with `cameraOnly()` I would like to be able to set ImagePicker to either A) only allow the front camera or B) start with the front camera and allow the user to switch to the rear camera.

**Describe the solution you'd like**
For either of the cases above, a config option similar to `cameraOnly` like `frontCameraOnly` or `startFrontCamera` that either restricts user to front camera, or starts them off with it. 

**Describe alternatives you've considered**
I tried overriding the provider methods in library, but haven't had much luck with it.

**Additional context**
Basically I'm using this library to capture or pick profile photos. Those are typically taken as selfies and not from the rear camera.

**Comment by ArisChoice** [comment]
Update below function in   **IntentUtils**  this class:-
This is working for me

```
/**
     * @return Intent Camera Intent
     */
    @JvmStatic
    fun getCameraIntent(uri: Uri, tryFrontCamera: Boolean): Intent {
        val intent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)

        if (tryFrontCamera) when {
            Build.VERSION.SDK_INT >= Build.VERSION_CODES.O -> {

                intent.putExtra(CAMERA_FACING_EXTRA, CameraCharacteristics.LENS_FACING_FRONT) // Tested on API 27 Android version 8.0(Nexus 6P)
                intent.putExtra("android.intent.extra.USE_FRONT_CAMERA", true)
                intent.putExtra("android.intent.extras.CAMERA_FACING", CameraCharacteristics.LENS_FACING_FRONT)
                intent.putExtra("android.intent.extras.CAMERA_FACING", android.hardware.Camera.CameraInfo.CAMERA_FACING_FRONT);
            }
            Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1 && Build.VERSION.SDK_INT < Build.VERSION_CODES.O -> {
                intent.putExtra(
                    CAMERA_FACING_EXTRA,
                    CameraCharacteristics.LENS_FACING_FRONT
                ) // Tested on API 24 Android version 7.0(Samsung S6)
            }
            Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP_MR1 -> intent.putExtra(
                CAMERA_FACING_EXTRA,
                1
            ) // Tested API 21 Android version 5.0.1(Samsung S4)
        }

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            intent.putExtra(MediaStore.EXTRA_OUTPUT, uri)
        } else {
            intent.putExtra(MediaStore.EXTRA_OUTPUT, uri)
        }

        return intent
    }
```

**Comment by Drjacky** [comment]
@ArisChoice Could you please create a pull request and I'll check and if it works, I'll merge it to the codebase. Thanks!

**Comment by Drjacky** [comment]
Thanks to @ArisChoice, https://github.com/Drjacky/ImagePicker/releases/tag/2.1.15

P.S Some devices still don't support defining front camera as default - Like OnePlus

**Comment by shobhik** [comment]
@Drjacky would you recommend I copy those Intent extras to use this, or somehow try to set `tryFrontCamera` by intercepting the provider? Not sure what the ideal usage would be. Also thanks @ArisChoice for the update!

**Comment by Drjacky** [comment]
@shobhik Pass `true` to `tryFrontCamera` parameter in `getCameraIntent`

**Comment by Ahsan2408** [comment]
where do I call this function please give me an example. I am trying to open the front camera at first.

**Comment by Drjacky** [comment]
@Ahsan2408 
https://github.com/Drjacky/ImagePicker/blob/54e52258fed1d3f7765a8382b4415c0e1912c79d/imagepicker/src/main/kotlin/com/github/drjacky/imagepicker/constant/ImageProvider.kt

```kotlin
ImagePicker.with(this)
    //...
    .provider(ImageProvider.BOTH) //Or bothCameraGallery()
    .createIntentFromDialog { launcher.launch(it) }
```
